docs/source/release/v6.6.0/Diffraction/Single_Crystal/New_features/34657.rst

```diff
@@ -0,0 +1,2 @@
+* Added new option to :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>`: to get intial TOF window from back-to-back exponential coeficients if sepcified in the insturment parameters.xml
+* Added back-to-back exponential coeficients to SXD parameters.xml file
```

Comment:
I think there is a couple of misspellings in these two lines, including 'coefficients'

Reply:
Thanks, I've corrected the spelling - the other release note file looked OK (checked in word this time!)